xarray/core/dataset.py

```diff
@@ -4032,6 +4193,26 @@ def expand_dims(
             coordinates. Note, this is an alternative to passing a dict to the
             dim kwarg and will only be used if dim is None.
 
+        Examples
+        --------
+
+        >>> dataset = xr.Dataset({"temperature": ([], 25.0)})
```

Comment:
Nice neat example.

I would add a second example with 1D or 2D data so that you can show what the `axis` argument does.